nss-day-cohort-48/the-worst-site-ever-rio-grandes
# Tlc philosophy

This is the gary busey philosophy page:

##to test it:
1. Open the page in Chrome.
2. Verify that all of the text is in a difficult to read color.
3. Verify that all the text is flashing.
4. Verify that two videos appear.
5. Verify that clicking on the videos starts play.
6. Verify that there are no controls for video.
7. Verify that clicking on the videos opens a new window. 
8. Open HTML code to make sure it makes sense.
9. Open philosophy.css file to make sure it makes sense.
